src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.html

```diff
@@ -1,249 +1,194 @@
-              <cd-select-badges [data]="userForm.controls.roles.value"
-                                [options]="allRoles"
-                                [messages]="messages"></cd-select-badges>
-            </span>
-          </div>
-        </div>
-
-        <!-- Enabled -->
-        <div class="form-group row"
-             *ngIf="!isCurrentUser()">
-          <div class="cd-col-form-offset">
-            <div class="custom-control custom-checkbox">
-              <input type="checkbox"
-                     class="custom-control-input"
-                     id="enabled"
-                     name="enabled"
-                     formControlName="enabled">
-              <label class="custom-control-label"
-                     for="enabled"
-                     i18n>Enabled</label>
-            </div>
-          </div>
-        </div>
-
-        <!-- Force change password -->
-        <div class="form-group row"
-             *ngIf="!isCurrentUser() && !authStorageService.isSSO()">
-          <div class="cd-col-form-offset">
-            <div class="custom-control custom-checkbox">
-              <input type="checkbox"
-                     class="custom-control-input"
-                     id="pwdUpdateRequired"
-                     name="pwdUpdateRequired"
-                     formControlName="pwdUpdateRequired">
-              <label class="custom-control-label"
-                     for="pwdUpdateRequired"
-                     i18n>User must change password at next logon</label>
-            </div>
-          </div>
-        </div>
+           class="form-header">{{ action | titlecase }} {{ resource | upperFirst }}
+      </div>
+      <!-- UserName -->
+      <div class="form-item">
+        <cds-text-label for="username"
+                        cdRequiredField="Username"
+                        [invalid]="!userForm.controls.username.valid && userForm.controls.username.dirty"
+                        [invalidText]="usernameError"
+                        i18n>Username
+          <input cdsText
+                 placeholder="Username..."
+                 i18n-placeholder
+                 id="username"
+                 name="username"
+                 formControlName="username"
+                 [invalid]="!userForm.controls.username.valid && userForm.controls.username.dirty"
+                 autofocus>
+        </cds-text-label>
+        <ng-template #usernameError>
+          <span *ngIf="userForm.showError('username', formDir, 'required')">
+            <ng-container i18n>
+             This field is required.
+            </ng-container>
+          </span>
+          <span *ngIf="userForm.showError('username', formDir, 'notUnique')">
+            <ng-container i18n>
+              The username already exists.
+            </ng-container>
+          </span>
+        </ng-template>
+      </div>
+      <!-- Password -->
+      <div class="form-item">
+        <cds-password-label for="password"
```

Comment:
I see we want to use higher version for using this. I think we can bump the version separately and then take this